### ISSUE TITLE
Beat Saber Update - Most Mods Broken

### ISSUE THREAD

**Issue by Sabore** [Assistant/ModAssistant#35]
After the recent Beat Saber update most mods are broken. It looks like it will still play custom songs that were downloaded, but finding new songs, custom sabers, avatars, camera, etc. no longer function. Any ETA on updated Mod Assistant that works with new version?

**Comment by Assistant** [comment]
Mod Assistant is unrelated to the Beat Saber version. It only installs the mods you tell it to install, it doesn't care about what version of the game you have, so no update is needed. 

Furthermore, the mods that worked in the previous version still work on the current version of Beat Saber.

**Comment by Sabore** [comment]
Interesting.

I had the game loaded and all mods were working correctly. I reloaded the game and it notified me of an update and I clicked continue without thinking. The mods stopped working immediately after the update. I've run Mod Assistant to verify everything is current. 

Do you suggest a clean install of the mods via Mod Assistant again? Any advice is appreciated.

Thanks.

**Comment by Assistant** [comment]
I suggest reading the support section of bsmg.wiki, to try and see what the issue is, but the most surefire way to fix all issues is to make a clean reinstall, meaning:

Uninstalling the game, the deleting the entire folder, then running the game once without any mods, and then installing mods.

**Comment by chendaniely** [comment]
@Sabore I think I was having the same issue as you. I ended up backup (i.e., moving) my beat saber installation and re-installing it (from steam) and re-installed all the Mods from Mod Assistant (v 1.0.25).
I still get a warning about a game update, which makes sense the game updated from version 1.1.0p1 to 1.2 but I was able to get my mods working again.

For songs. I personally use BeatDrop 2 separately for songs/playlists
